modules/dynamic/test/src-jvm/smithy4s/dynamic/model/JsonIOProtocol.scala

```diff
@@ -0,0 +1,83 @@
+package smithy4s
```

Comment:
should this go to shared sources instead of jvm-specific?

Reply:
Yup, you are correct